### PR TITLE
Fixes broken link.

### PR DESCRIPTION
**Pull request by pierreozoux**
Closes #3120
/assign @cblecker

### CODE REVIEW

**Comment by mrbobbytables** [comment]
Thanks @pierreozoux! :) 

/ok-to-test

File: generator/list.tmpl [hunk 1]
```diff
@@ -9,7 +9,7 @@ depending on their needs and workflow.
 
 Each group's material is in its subdirectory in this project.
 
-When the need arises, a [new SIG can be created](sig-creation-procedure.md)
+When the need arises, a [new SIG can be created](sig-governance.md#sig-creation-and-maintenance-procedure)
```
Comment (nikhita) [inline generator/list.tmpl]: @pierreozoux thanks! you'll need to run `make generate` for the tests to pass :)

Comment (pierreozoux) [reply generator/list.tmpl]: can't this `make generate` be part of a pipeline? I'd love to help you build this pipeline, but I don't have the time at the moement :/

Comment (mrbobbytables) [reply generator/list.tmpl]: @pierreozoux The sig-list.md file is intended to be tracked in git and not an artifact stored some place else, it does make sense to run it before committing.

I completely forgot about running `make generate` to ensure the synced changes when updating the template. My apologies. I should have mentioned that in the other issue. >_<

**Comment by spiffxp** [comment]
Hi @pierreozoux do you think you'll have time to rebase this PR and run `make generate`?

**Comment by nikhita** [comment]
Fixed in https://github.com/kubernetes/community/pull/3403

/close

**Pull request closed without merging**